### DISCUSSION TITLE
Un topweekserver

### DISCUSSION THREAD

**Discussion by symsym-1629** [Crownicles/Crownicles#1099] [Idées acceptées / déjà implémentées]
Bouh ! Suggestion de la semaine plop. Je verrais bien cela ds le jeu car ça implémenterai une fonctionnalité cool qui permettrait notamment à certains serveurs communautaires faisant des giveaway d'utiliser le bot et donc d'amener des joueurs et un autre type de communauté.

**Comment by romain22222** [comment]
perso je trouve ca super intéressant, upvoted

**Comment by leuhvraiminoryo** [comment]
excellente idée ! je supporte aussi

**Comment by BastLast** [comment]
Je suis pas contre cette idée mais je pense qu'il faudra en rediscuter quand on fera le retravaille de la commande top

En fait j'aimerais que les différents top soit accessible avec un système de réaction ou de boutons

**Reply by BastLast** [reply]
Sinon pour parler de l'idée en elle-même je pense que ça peut être sympa en effet mais pas ultra prio par contre du coup on va refaire la commande top de 0 très très bientôt en soit ca peut être l'occasion...

**Reply by symsym-1629** [reply]
👍

**Reply by beaud37210** [reply]
Sinon tu peux essayer avec un système de liste non?

**Comment by beaud37210** [comment]
Eh t'aurai pas trouvé ça sur le message de @beaud37 dans #detente-media?(mon message) c'était mon idée de base😅

**Reply by BastLast** [reply]
50000 personnes ont suggéré ça

**Reply by beaud37210** [reply]
Ow

**Reply by symsym-1629** [reply]
Naope je l'avais déjà en tête

**Comment by Pietagorh** [comment]
Du coup ça a été accepté

**Reply by BastLast** [reply]
En effet, cette suggestion est dans la dernière mise à jour.
